config/default.go

```diff
@@ -38,7 +38,9 @@ func NewConfig() *Config {
 				"-H", DOCKER_SYSTEM_HOST,
 			},
 		},
-		Modules: []string{},
+		Modules: []string{
```

Comment:
I know @sheng-liang asked for this, but I disagree.  I think we should just compile tun into the default kernel.  I'm pretty sure if you were to swap out a kernel that didn't have tun, then our default configuration will fail because it can't load tun.